internal/clients/team3/judge.go

```diff
@@ -11,4 +14,139 @@ type judge struct {
 	c *client
 }
 
-// Override functions here, see president.go for examples
+// PayPresident pays the president's salary
+func (j *judge) PayPresident(salary shared.Resources) (shared.Resources, bool) {
+	j.c.params.salaryThreshold = 150 //parameter that we need to define
+
```

Comment:
Once PR #208 is merged, we won't be able to keep the part of the salary that we decide not to pay the President.

Reply:
Okay, will revert to base implementation or as discussed we can still make a partial payment but we just won't keep the deducted amount in our own local pool.